guides/image_classification_in_tensorflow.md

```diff
@@ -0,0 +1,88 @@
+# Image Classification in TensorFlow and Keras
+
+related_spaces: abidlabs/tensorflow-image-classifier
```

Comment:
This space doesn't exist. also iframing a 400 error below

please also include the full link when you correct it, as well as more related spaces like:
* https://huggingface.co/spaces/keras-io/keras-image-classifier
* https://huggingface.co/spaces/keras-io/Image-classification-with-EANet